distribution/src/main/resources/VERSIONS.md

```diff
@@ -23,19 +23,20 @@ This file lists the recommended components for this Service Pack. Use it along w
 ± Based on Alfresco Community Repository @dependency.alfresco-community-repo.version@
 
 ##### Alfresco Modules and Integrations
-| Component | Recommended | Also Available |
-|---|---|---|
-| Alfresco Share Services | @dependency.alfresco-community-repo.version@ |
-| Alfresco Google Docs Integration | @alfresco.googledrive.version@ |
-| Alfresco Governance Services (Repository) | @dependency.alfresco-enterprise-repo.version@ |
-| Alfresco Governance Services (Share) | @dependency.alfresco-enterprise-share.version@ |
-| Alfresco Intelligence Services | @alfresco.ais.version@ |
-| Alfresco Office Services | @alfresco.aos-module.version@ |
-| Alfresco Content Connector for Salesforce | @alfresco.salesforce-connector.version@ |
-| Alfresco S3 Connector | @alfresco.s3connector.version@ |
-| Alfresco Content Connector for Azure | @alfresco.azure-connector.version@ |
-| Alfresco Outlook Integration | @alfresco.outlook.version@ |
-| Alfresco SDK | 5.0 | 4.2 |
+| Component                                        | Recommended                                    | Also Available |
+|--------------------------------------------------|------------------------------------------------|----------------|
+| Alfresco Share Services                          | @dependency.alfresco-community-repo.version@   |
+| Alfresco Google Docs Integration                 | @alfresco.googledrive.version@                 |
+| Alfresco Governance Services (Repository)        | @dependency.alfresco-enterprise-repo.version@  |
+| Alfresco Governance Services (Share)             | @dependency.alfresco-enterprise-share.version@ |
+| Alfresco Intelligence Services                   | @alfresco.ais.version@                         |
+| Alfresco Office Services                         | @alfresco.aos-module.version@                  |
+| Alfresco Content Connector for Salesforce        | @alfresco.salesforce-connector.version@        |
+| Alfresco S3 Connector                            | @alfresco.s3connector.version@                 |
+| Alfresco Content Connector for Azure             | @alfresco.azure-connector.version@             |
+| Alfresco Outlook Integration                     | @alfresco.outlook.version@                     |
+| Alfresco Extension for Hyland Experience Insight | @alfresco.hxinsight-connector.version@         |
```

Comment:
I'm not sure it matters, but I think the official name is something like `Hyland Experience Insight Alfresco Connector`. Alternatively we generally call it `Alfresco Hyland Experience Insight Connector` or `Alfresco Hx Insight Connector`. Is the name you picked copied from somewhere?

Reply:
I think it can matter for PMs

Reply:
Nope - it's my invention. I wasn't sure how to name it, because I though that under the term `Hyland Experience Insight Alfresco Connector` is hidden more components, where only one of which (JAMP, extension) is mentioned here 🤔

Reply:
I don't mind to change the name